Source/Relativity.Test.Helpers/Relativity.Test.Helpers/HttpRequestHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Relativity.Test.Helpers.SharedTestHelpers;
+
+namespace Relativity.Test.Helpers
+{
+	public class HttpRequestHelper<T>
```

Comment:
we need to add an interface for this class and remove static implementations. Doing this refactor will help with writing unit tests.